encoders/firebase-decoders-json/firebase-decoders-json.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
     testImplementation "com.google.truth:truth:$googleTruthVersion"
     testImplementation 'org.mockito:mockito-core:2.25.0'
     implementation project(path: ':encoders:firebase-encoders-json')
+    implementation project(path: ':encoders:firebase-encoders-processor:test-support')
```

Comment:
you should not use this dependency, it's only needed to be able to test encoders-processor, depending on `:encoders:firebase-encoders-json` should be enough(which you already do).